exchange/exchange.go

```diff
@@ -163,8 +163,8 @@ func (e *exchange) HoldAuction(ctx context.Context, r AuctionRequest, debugLog *
 
 	requestDebugInfo := getDebugInfo(r.BidRequest, requestExt)
 
-	debugInfo := requestDebugInfo && r.Account.DebugAllow
-	debugLog.Enabled = debugLog.Enabled && r.Account.DebugAllow
+	debugInfo := debugLog.DebugOverride || requestDebugInfo && r.Account.DebugAllow
```

Comment:
Why do you have () in 167, but not 166? If I am parsing it properly, I think the logic works out the same either way, but it is confusing as it looks like the logic should be different in the two cases.

Reply:
Yes, good catch, fixed.